collect_app/src/main/java/org/odk/collect/android/formentry/questions/WidgetViewUtils.java

```diff
@@ -19,13 +20,22 @@
 import org.odk.collect.android.widgets.interfaces.ButtonWidget;
 
 import static android.view.View.GONE;
+import static org.odk.collect.android.utilities.ViewUtils.dpFromPx;
+import static org.odk.collect.android.widgets.StringWidget.FIELD_HORIZONTAL_MARGIN_MODIFIER;
 
 public class WidgetViewUtils {
 
     private WidgetViewUtils() {
 
     }
 
+    public static int getStandardMargin(Context context) {
+        Resources resources = context.getResources();
+        int marginStandard = dpFromPx(context, resources.getDimensionPixelSize(R.dimen.margin_standard));
+
+        return marginStandard - FIELD_HORIZONTAL_MARGIN_MODIFIER;
```

Comment:
Would be good to move this contstant to this file and name it something like `WIDGET_ANSWER_STANDARD_MARGIN_MODIFIER`.